src/sparseml/pytorch/torchvision/train.py

```diff
@@ -443,7 +476,7 @@ def collate_fn(batch):
             evaluate(model, criterion, data_loader_test, device=device)
         return
 
-    manager = ScheduledModifierManager.from_yaml(args.recipe_path)
+    # TODO do we still need this in case of resume or checkpoint_path?
```

Comment:
in resume, prefer to use initialize (modify ok if it's just initializing, check code)

Reply:
I kept this in - otherwise the optimizer isn't changed without this call?